plugins/generator-addon-1.20.x/addon-1.20.x/templates/loottable.json.ftl

```diff
@@ -100,6 +100,22 @@
                       "formula": "minecraft:ore_drops"

```

Comment:
Did you test in bedrock edition too?

Reply:
I did not, I will.  It is my understanding that the loottable json is the same between bedrock and Java though

Reply:
While it is quite similar, it is not entirely the same, this is why it is important to test